mpas_analysis/ocean/ocean_modelvsobs.py

```diff
@@ -136,28 +152,22 @@ def ocn_modelvsobs(config, field):
             # Increment month value to be consistent with the model output
             dsObs.iMONTH.values += 1
 
-            # Rename the dimensions to be consistent with other obs. data sets
-            dsObs.rename({'month': 'calmonth', 'lat': 'latCoord',
-                          'lon': 'lonCoord'}, inplace=True)
-            dsObs.rename({'iMONTH': 'Time', 'iLAT': 'lat', 'iLON': 'lon'},
-                         inplace=True)
+            dsObs.rename({'iMONTH': 'Time'}, inplace=True)
             # set the coordinates now that the dimensions have the same names
-            dsObs.coords['lat'] = dsObs['latCoord']
-            dsObs.coords['lon'] = dsObs['lonCoord']
-            dsObs.coords['Time'] = dsObs['calmonth']
-            dsObs.coords['month'] = ('Time', np.array(dsObs['calmonth'], int))
+            dsObs.coords['lat'] = dsObs['lat']
+            dsObs.coords['lon'] = dsObs['lon']
+            dsObs.coords['Time'] = dsObs['month']
+            dsObs.coords['month'] = ('Time', np.array(dsObs['month'], int))
             # no meaningful year since this is already a climatology
             dsObs.coords['year'] = ('Time', np.ones(dsObs.dims['Time'], int))
 
             dsObs = mpas_xarray.subset_variables(dsObs, [obsFieldName,
                                                          'month'])
-            # Reorder dataset for consistence with other obs. data sets
-            dsObs = dsObs.transpose('Time', 'lat', 'lon')
```

Comment:
Hmm, so xarray is supposed to be smart enough that the order of indices doesn't matter.  But I think this only works because these data sets get remapped and the result after remapping is in the right order.  Maybe it won't work if one picks a comparison grid that has the same resolution as the observations (1x1 degree).

To be honest, I think the best solution would be to pre-process these observations in a separate script to put them on the default comparison grid, and to make sure they are transposed properly.  Then, we wouldn't have this issue here.

For now, I think this change is okay, but I'll revert it if you're concerned.